retrieval_service/datastore/providers/cloudsql_mysql_test.py

```diff
@@ -0,0 +1,646 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+from datetime import datetime
+from typing import Any, AsyncGenerator, List
+
+import pymysql
+import pytest
+import pytest_asyncio
+from csv_diff import compare, load_csv  # type: ignore
+from google.cloud.sql.connector import Connector
+
+import models
+
+from .. import datastore
+from . import cloudsql_mysql
+from .test_data import (
+    amenities_query_embedding1,
+    amenities_query_embedding2,
+    foobar_query_embedding,
+    policies_query_embedding1,
+    policies_query_embedding2,
+)
+from .utils import get_env_var
+
+pytestmark = pytest.mark.asyncio(scope="module")
+
+
+@pytest.fixture(scope="module")
+def db_user() -> str:
+    return "mysql"
+
+
+@pytest.fixture(scope="module")
+def db_pass() -> str:
+    return "my-cloudsql-pass"
+
+
+@pytest.fixture(scope="module")
+def db_project() -> str:
+    return "juliaofferman-playground"
+
+
+@pytest.fixture(scope="module")
+def db_region() -> str:
+    return "us-central1"
+
+
+@pytest.fixture(scope="module")
+def db_instance() -> str:
+    return "my-vector-app"
```

Comment:
I am assuming we want to update these...

Reply:
Done